src/main/java/io/eroshenkoam/xcresults/export/ExportCommand.java

```diff
@@ -139,6 +144,17 @@ private void runUnsafe() throws Exception {
         }
     }
 
+    private ExportMeta getTestMeta(ExportMeta meta, JsonNode testableSummary) {
```

Comment:
Ensure suite name is captured and exported as part of JSON so that allure report when generate, the test will be categorised by suite